### PR TITLE
Few bug fixes and doc enhancements.

### PR DESCRIPTION
**Pull request by jeffkala**

### CODE REVIEW

**Comment by jeffkala** [comment]
fixes #40

**Comment by jeffkala** [comment]
@jvanderaa this should cover the 3 issues mentioned.  Can't seem to figure out why updating the lock file isn't finishing in my case.  Similar to why the pipeline is failing on black, when I run black locally its passing.

File: nautobot_chatops_ansible/tower.py [hunk 1]
```diff
@@ -31,7 +30,7 @@ def __init__(
             verify_ssl (bool): Verify SSL connections. Defaults to True.
         """
         if tower_uri:
-            self.uri = tower_uri.rstrip()
+            self.uri = tower_uri.rstrip().rstrip("/")
```
Comment (jvanderaa) [inline nautobot_chatops_ansible/tower.py:33]: Should we also make sure that it has the appropriate `http://` here? Based on a different conversation lately about host/uri/url

**Review by jvanderaa — APPROVED** [review]

File: nautobot_chatops_ansible/tower.py (hunk 1 above)
Comment (jeffkala) [reply nautobot_chatops_ansible/tower.py]: add this to the latest commit

**Review by jvanderaa — APPROVED** [review]

**Comment by jvanderaa** [comment]
@jeffkala - tests

**Comment by jeffkala** [comment]
> @jeffkala - tests

@jvanderaa all my test pass locally.  Can you restart the pipeline?  Might have failed during some of the recent github issues.

**Comment by jvanderaa** [comment]
Thanks @jeffkala

**Pull request merged**